src/ml-engine/main_app/infrastructure/defi_llama.py

```
@@ -0,0 +1,62 @@
+import requests
+import pandas as pd
+
+def get_pool_summary():
+    url = f"https://yields.llama.fi/pools"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)